File: C/medidas.c
#include <stdio.h>

int main(){
    double a, b, c, areaQuadrado, areaTriangulo, areaTrapezio;

    printf("Digite a medida A: ");
    scanf("%lf", &a);

    printf("Digite a medida B: ");
    scanf("%lf", &b);

    printf("Digite a medida C: ");
    scanf("%lf", &c);

    areaQuadrado = a * a;
    areaTriangulo = (a * b) / 2;
    areaTrapezio = ((a + b) * c) / 2;

    printf("AREA DO QUADRADO = %.4lf\n", areaQuadrado);
    printf("AREA DO TRIANGULO = %.4lf\n", areaTriangulo);
    printf("AREA DO TRAPEZIO = %.4lf\n", areaTrapezio);
    return 0;
}

File: Java/medidas.java
import java.util.Locale;
import java.util.Scanner;

public class medidas {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		double a, b, c, quadrado, triangulo, trapezio;

		System.out.print("Digite a medida A: ");
		a = sc.nextDouble();
		System.out.print("Digite a medida B: ");
		b = sc.nextDouble();
		System.out.print("Digite a medida C: ");
		c = sc.nextDouble();

		quadrado = Math.pow(a, 2);
		triangulo = (a * b) / 2;
		trapezio = ((a + b) * c) / 2;

		System.out.printf("AREA DO QUADRADO = %.4f\n", quadrado);
		System.out.printf("AREA DO TRIANGULO = %.4f\n", triangulo);
		System.out.printf("AREA DO TRAPEZIO = %.4f\n", trapezio);

		sc.close();
	}
}
